packages/block-editor/src/components/inserter/index.js

```diff
@@ -149,40 +149,58 @@ class Inserter extends Component {
 
 		if ( isQuick ) {
 			return (
-				<QuickInserter
-					onSelect={ ( blocks ) => {
-						const firstBlock =
-							Array.isArray( blocks ) && blocks?.length
-								? blocks[ 0 ]
-								: blocks;
-						if (
-							onSelectOrClose &&
-							typeof onSelectOrClose === 'function'
-						) {
-							onSelectOrClose( firstBlock );
-						}
+				<>
+					<QuickInserter
+						onSelect={ ( blocks ) => {
+							const firstBlock =
+								Array.isArray( blocks ) && blocks?.length
+									? blocks[ 0 ]
+									: blocks;
+							if (
+								onSelectOrClose &&
+								typeof onSelectOrClose === 'function'
+							) {
+								onSelectOrClose( firstBlock );
+							}
+							onClose();
+						} }
+						rootClientId={ rootClientId }
+						clientId={ clientId }
+						isAppender={ isAppender }
+						prioritizePatterns={ prioritizePatterns }
+						selectBlockOnInsert={ selectBlockOnInsert }
+					/>
+					<Button
```

Comment:
@getdave Do you think you can move this button to appear in the DOM right above the search input? Other than that, this works amazing!

Reply:
I looked at this but the problem is that the `Close` button becomes the first focusable element and thus is auto-focused when the inserter opens. This provides poor UX for visual users so we'll need to figure out an alternative visual display for this close UI.

@SaxonF I wonder if you have any thoughts on using a `X` close icon on these popovers which we could use consistently across all popovers? Something universal would be better than adding them to all UIs individually.